font.css

```diff
@@ -0,0 +1,21 @@
+@font-face {
+    font-family: 'PT Serif', serif;
+    src: url('font/PTSerif-Bold.ttf');
+    font-weight: bold;
+    font-style: normal;
+}
+
+@font-face {
+    font-family: 'Roboto', sans-serif;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/myfinale/pullRequest?prid=5606941)